Modules/Core/Transform/test/itkEuler3DTransformGTest.cxx

```diff
@@ -33,3 +33,20 @@ TEST(Euler3DTransform, SetFixedParametersThrowsWhenSizeIsLessThanInputSpaceDimen
     ASSERT_THROW(transform->SetFixedParameters(fixedParameters), itk::ExceptionObject);
   }
 }
+
+TEST(Euler3DTransform, TestSetGetCenterAfterSetIdentity)
+{
+  using EulerTransformType = itk::Euler3DTransform<>;
+
+  auto eulerTransform = EulerTransformType::New();
```

Comment:
Just a nitpick here: Looks like `eulerTransform` is unused, and may be removed!

Reply:
Not a nitpick.  It I'm trying to squeeze this between a huge pile of grant reviews.  I copy-n-pasted when moving to GTest.

Reply:
Fixed.